PXWeb/Selection.aspx.cs

```diff
@@ -423,8 +423,49 @@ private void InitializeMetadata(string path)
                 bool bMeta = PXWeb.Settings.Current.Database[url.Database].Metadata.UseMetadata;
             }
 
+
+           SetBulkLink();
+
         }
 
+        private void SetBulkLink()
```

Comment:
Should we also do a check to verify that the bulk file actually exists. If the Bulk file do not exist then we should probably not show the link